kolibri/plugins/user/assets/src/views/UserProfileSideNavEntry.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-
+  <!-- eslint-disable kolibri/vue-no-undefined-string-uses -->
```

Comment:
what's wrong with this file?

Reply:
The linter wasn't pulling $trs defined on a couple of components that were defined as variables and then exported later on - it was only finding the definitions in components defined as part of the `export default` declaration.

I've refactored it to fix this and have removed these comments.

cc @indirectlylit @jonboiser